### PR TITLE
LPS-167001 Integration tests and use new validate

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-167001

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 1]
```diff
@@ -2501,6 +2509,47 @@ private Map<String, Serializable> _getValuesFromDatabase(
...
+			objectEntry.getObjectEntryId());
+
+		Assert.assertEquals(
+			"listTypeEntryKey1, listTypeEntryKey2",
+			values.get("multipleListTypeEntriesKey"));
+
+		_objectEntryLocalService.deleteObjectEntry(
+			objectEntry.getObjectEntryId());
+	}
+
```
Comment (gabrielwas) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:2564]: Can you adapt this code and put it under the testUpdateObjectEntry method?

There, you'll have to:

1. Add a new multiselect field here: 

https://github.com/liferay/liferay-portal/blob/e573e6fefc0c22989f3e5c2930f3d4ac67c9539f/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java#L188-L249

2. Add new checks here:

https://github.com/liferay/liferay-portal/blob/e573e6fefc0c22989f3e5c2930f3d4ac67c9539f/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java#L2012-L2030

and here:

https://github.com/liferay/liferay-portal/blob/e573e6fefc0c22989f3e5c2930f3d4ac67c9539f/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java#L2077-L2099

And feel free to adapt/create code for the needs you want. Thx

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java [hunk 2]
```diff
@@ -100,6 +102,11 @@ public class ObjectFieldLocalServiceTest {
 
 	@Before
 	public void setUp() throws Exception {
+		PropsUtil.addProperties(
+			UnicodePropertiesBuilder.setProperty(
+				"feature.flag.LPS-164278", "true"
+			).build());
+
```
Comment (gabrielwas) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:111]: Can you put this just surrounding the code you added? Thx

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 3]
```diff
@@ -584,6 +597,13 @@ public void testAddObjectEntry() throws Exception {
 					"\"listTypeEntryKeyRequired\"",
 				objectEntryValuesException.getMessage());
 		}
+
+		_testObjectEntryWithMultiselectPicklist();
+
+		PropsUtil.addProperties(
+			UnicodePropertiesBuilder.setProperty(
+				"feature.flag.LPS-158776", "false"
+			).build());
```
Comment (gabrielwas) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:606]: Can you put this in the tearDown method?

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java [hunk 4]
```diff
@@ -194,6 +203,25 @@ public void testAddCustomObjectField() throws Exception {
...
+			).name(
+				"a" + RandomTestUtil.randomString()
+			).build());
+
+		PropsUtil.addProperties(
+			UnicodePropertiesBuilder.setProperty(
+				"feature.flag.LPS-164278", "false"
+			).setProperty(
+				"feature.flag.LPS-158776", "false"
+			).build());
```
Comment (gabrielwas) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:224]: Can you put this in the tearDown method?

**Review by gabrielwas — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java (hunk 1 above)
Comment (p-albuquerque) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:2564]: copy that

Comment (p-albuquerque) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:2564]: The first point I had already done

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
resent here https://github.com/liferay-objects/liferay-portal/pull/1309

**Pull request closed without merging**